### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by mtrejor** [karan/atom-terminal#26]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/usr/share/atom/resources/app/src/atom.js:54:11)
Object.activate (/home/marcos/.atom/packages/atom-terminal/lib/atom-terminal.coffee:45:13)
```

**Comment by tkw1536** [comment]
See Issue #22. This has been fixed, but a new version has yet to be published (see #25 ). Temporary workaround is to via command line (https://github.com/karan/atom-terminal/issues/22#issuecomment-75056194)

**Comment by karan** [comment]
Fixed and new version pushed.
